code/modules/admin/verbs/adminhelp.dm

```diff
@@ -20,18 +20,21 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
 	adminhelped = 1 //Determines if they get the message to reply by clicking the name.
 
 	var/msg
-	var/list/type = list ("Suggestion / Bug Report", "Gameplay / Roleplay Issue")
+	var/list/type = list ("Suggestion / Bug Report", "Gameplay / Roleplay Issue", "Admins Spawn Shit")
 	var/selected_type = input("Pick a category.", "Admin Help", null, null) as null|anything in type
 	if(selected_type == "Gameplay / Roleplay Issue")
 		msg = input("Please enter your message:", "Admin Help", null, null) as message|null
 
 	if(selected_type == "Suggestion / Bug Report")
-		switch(alert("Adminhelps are not for suggestions or bug reports - they should be posted on our Gitlab.",,"Go to Gitlab","Cancel"))
-			if("Go to Gitlab")
-				src << link("https://gitlab.com/cmdevs/ColonialMarines/issues")
+		switch(alert("Adminhelps are not for suggestions or bug reports - they should be posted on our Github. #WHENYOUCODEIT",,"Go to Gitlab","Cancel"))
+			if("Go to Github")
+				src << link("https://github.com/ColonialMarines-Mirror/ColonialMarines-2018/issues")
```

Comment:
The github is not for suggestions either, that's the forums

Reply:
Right, lemme fix that